### PR TITLE
Release/v 1.0

### PR DESCRIPTION
**Pull request by jssol**
#### APIs

- First, you need to find an API so you can base the development of the webapp around it. The API should allow you to:
  - Get a list of items with a unique item id (or generate the unique id).
  - For a given item, get detailed information about it.
- Use our [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to record the different user interactions (likes, comments).

#### Interfaces

- [x] Built these interfaces:
  - The home page.
  - The comments popup.

- [x] When the page loads, the web app retrieves data from:
  - The selected API shows the list of items on the screen.
  - The Involvement API to show the item likes.
  - When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
  - When the user clicks on the "Comments" button, the Comments popup appears.

- [x] Comments popup
  - When the popup loads, the web app retrieves data from:
  - The selected API shows details about the selected item.
  - The Involvement API to show the item comments.
  - When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.

#### Counters

Counters We have counters in all the interfaces that show:

- The number of items (home).
- The number of comments (comments popup).

### Useful links
- [Full requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)
- [Live Demo](https://github.com/jssol/api-based-webapp/pull/46)

### CODE REVIEW

**Review by Ismailco — APPROVED** [review]
Nicely Done 👏🏼
However, you can remove ``The number of reservations (reservations popup) - only for the groups of 3 students.`` (from the PR description) since we don't have a third team member.

Great work @jssol 💯

**Comment by jssol** [comment]
> Nicely Done 👏🏼 However, you can remove `The number of reservations (reservations popup) - only for the groups of 3 students.` (from the PR description) since we don't have a third team member.
> 
> Great work @jssol 100

Let's ask for a review :tada:

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+    comments.forEach(() => {
+      count += 1;
+    });
+  } else {
+    count = 0;
+  }
+  return count;
+};
+
+const showComments = async (id) => {
```
Comment (hiromataba) [inline src/index.js:99]: - [ ] [OPTIONAL] Please update  the comments on the screen after submitting them, for now it is now updating the screen unless you close it and then reopen it

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada:  :champagne:  :confetti_ball: 
Hi @jssol and @Ismailco :wave: :wave:

200w 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Awesome design :heavy_check_mark: 
- Your README file looks very professional :+1: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (jssol) [reply src/index.js:99]: We will be working on this very soon thanks :pray:

**Pull request merged**